Jquery_table.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>jquery-table</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <style type="text/css">
+        table, tr, td, th {
+            border: 1px solid black;
+        }
+        th {
+            width: 65px;
+        }
+    </style>
+</head>
+
+<body>
+    <p id="demo"></p>
+    <p id="demo10"></p>
+    <p id="demo15"></p>
+    <button id="load">Load more</button>
+
+    <script type="text/javascript">
+        $(function () {
+            var students = [
+                [1, 'Giang', 'PHP2008', 6],
+                [2, 'Kien', 'PHP2104', 7],
+                [3, 'Nam', 'PHP2105', 8],
+                [4, 'Hieu', 'PHP2103', 8],
+                [5, 'Dat', 'PHP2103', 6],
+                [6, 'Hoang', 'PHP2103', 8],
+                [7, 'Hoan', 'PHP2103', 9],
+                [8, 'Khiem', 'PHP2103', 8],
+                [9, 'Quan', 'PHP2103', 7],
+                [10, 'Truong', 'PHP2103', 9],
+                [11, 'Huy', 'PHP2103', 9],
+                [12, 'Hung', 'PHP2103', 9],
+                [13, 'Tan', 'PHP2103', 8],
+                [14, 'Chien', 'PHP2103', 7],
+                [15, 'Khang', 'PHP2103', 8],
+            ];
+
+            var studentLen = students.length;
+            var i=0;
+            var text = "<table width='100%'>"
+                + "<tr>"
+                    + "<th>No</th>"
+                    + "<th>Name</th>"
+                    + "<th>Class</th>"
+                    + "<th>Point</th>"
+                + "</tr>";
+
+            for (i; i < 5; i++) {
+                text += "<tr>";
+                
+                childArr = students[i];
+                childArrLen = childArr.length;
+            
+                for (var j = 0; j < childArrLen; j++) {
+                    text += "<td>" + childArr[j] + "</td>";
+                }
+                text += "</tr>";
+            }
+            console.log(i);
+            text += "</table>";
+            $('#demo').html(text);
+
+            $('#load').click(function(){
+                var text = "<table width='100%'>"
```

Comment:
thế này là bị in ra nhiều table mất rùi, thử update lại in ra 1 table xem sao e ơi